ledger-tool/src/main.rs

```diff
@@ -1065,6 +1065,13 @@ fn main() {
         .takes_value(true)
         .default_value(SnapshotVersion::default().into())
         .help("Output snapshot version");
+    let debug_keys_arg = Arg::with_name("debug_keys")
+        .long("debug-keys")
+        .validator(is_pubkey)
+        .value_name("ADDRESS")
+        .multiple(true)
```

Comment:
I think I'd lean to changing this cli arg (and text/var names) from `debug-keyS` to `debug-key`, since it can be provided multiple times, and one key per arg.